src/components/flowchart/index.js

```diff
@@ -418,7 +418,19 @@ export class FlowChart extends Component {
    */
   handleNodeMouseOver = (event, node) => {
     this.props.onToggleNodeHovered(node.id);
-    this.showTooltip(event, node);
+    node && this.showTooltip(event, node.fullName);
+  };
+
+  handleParamsIndicatorMouseOver = (event, node) => {
+    const parameterNames = this.props.nodesWithInputParams[node.id];
+    if (parameterNames) {
+      const label =
+        parameterNames.length > 1
+          ? `Parameters:${parameterNames.length}`
+          : parameterNames[0];
+      this.showTooltip(event, label);
```

Comment:
please set tests to test the different display options for the parameter nodes under `flowchart.test.js`.

Reply:
@studioswong -- thanks for flagging this. New tests have been added -- the function logic has been extracted to another function.